packages/sources/dxfeed-secondary/test/integration/adapter.test.ts

```diff
@@ -4,16 +4,33 @@ import * as process from 'process'
 import { server as startServer } from '../../src'
 import * as nock from 'nock'
 import * as http from 'http'
-import { mockPriceResponse } from './fixtures'
 import { AddressInfo } from 'net'
+import {
+  mockFirstHeartbeatMsg,
+  mockHandshake,
+  mockHeartbeatMsg,
+  mockPriceResponse,
+  mockSubscribe,
+  mockUnsubscribe,
+} from './fixtures'
+import {
+  mockWebSocketProvider,
+  mockWebSocketServer,
+  MockWsServer,
+  mockWebSocketFlow,
+} from '@chainlink/ea-test-helpers'
+import { WebSocketClassProvider } from '@chainlink/ea-bootstrap/dist/lib/middleware/ws/recorder'
+
+const sleep = (ms: number) => {
```

Comment:
You can use the new `sleep` util that Alejo abstracted to make this more DRY :man_dancing: 

```
import { util } from '@chainlink/ea-bootstrap'
await util.sleep(1000)
```